pages/_app.js

```diff
@@ -34,6 +34,14 @@ export default function App({ Component, pageProps }) {
       setCountries([...countries, { id, isFavourite: true }]);
     }
   }
+  function deleteImage(updatedCountry) {
+    setCountry(updatedCountry);
```

Comment:
if you already have a state "countries" why do you need a second state called "country"? Couldn't you just find the respective country inside the countries array?

Reply:
That was the only way  it worked. So I have the current page object in the [detailCountry] but it was not possible for me to give it to the [detailPictureView]. Is there a way to solve this?

Reply:
Maybe you can just rename it to "currentCountry" or second state? However, I don't feel confident enough to argue...